src/formDemoMockResponse.js

```diff
@@ -290,7 +290,7 @@ const metadata = {
       'refEntity': {
         'href': '/api/v1/it_emx_datatypes_TypeTestRef/meta',
         'hrefCollection': '/api/v1/it_emx_datatypes_TypeTestRef',
-        'idAttribute': 'id',
+        'idAttribute': 'value',
```

Comment:
why ?

Reply:
TypeTestRef testing objects (and real data as well) all have 'value' as ID attribute
this change makes it consistent with the rest of our testing data